server/service/src/apis/permissions.rs

```diff
@@ -198,6 +198,7 @@ pub enum Permissions {
     HISAddProcedure,
     ConfirmInternalOrderSent,
     ColdChainApi,
+    CentralServerAdmin,
```

Comment:
since we plan to just replace the central server with omSupply central server do we need two separate permissions? central server and omSupply central server are basically the same thing

Reply:
I think it should maybe be `EditCentralServerData` I think that would be clearer...